server/src/main/java/org/solarbank/server/dto/ErrorResponse.java

```diff
@@ -0,0 +1,32 @@
+package org.solarbank.server.dto;
+
+import lombok.Data;
+
+@Data
+public class ErrorResponse {
```

Comment:
We probably should add the `jackson`  serialisation strategy here. See my comment in `UserInputDto`.

Reply:
added annotation for deserialising JSON